devtools/platform-descriptor-json/pom.xml

```diff
@@ -65,6 +65,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>generate-gradle-wrapper</id>
+                        <phase>prepare-package</phase>
+                        <configuration>
+                            <executable>gradle</executable>
+                            <arguments>
+                                <argument>wrapper</argument>
+                            </arguments>
```

Comment:
Two comments here. I think we should use `./gradlew`. Maybe we could reuse the one in `devtools/gradle`. We would need to support both Unix-based OSes and Windows (so use the `gradlew.bat`).

Also, I think we should specify the version by using the `gradle-wrapper.version` property defined in `build-parent/pom.xml`.